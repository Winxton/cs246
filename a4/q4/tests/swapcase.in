w_mixedinput.txt swapcase
w_mixedinput.txt swapcase swapcase
