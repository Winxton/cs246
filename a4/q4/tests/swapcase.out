HELLO123
%!@#$%
ZEBRA
ABCDEFGHIJKLMNOPQRSTUVWXYZ
abcdefghijklmnopqrstuvwxyz
hello123
%!@#$%
zebra
abcdefghijklmnopqrstuvwxyz
ABCDEFGHIJKLMNOPQRSTUVWXYZ
